### PR TITLE
PP-5605: prepare configuration

### PR DESCRIPTION
**Pull request by jankowiakmaria**
* LedgerUrl config
* Add simple rest client factory
* Add leger service
* Update README with new environment variables

`LEDGER_URL` is already exposed (see connector config in infra repo)

### CODE REVIEW

File: src/main/java/uk/gov/pay/connector/app/RestClientFactory.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import javax.net.ssl.SSLContext;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+import static java.lang.String.format;
+
+public class RestClientFactory {
```
Comment (kbottla) [inline src/main/java/uk/gov/pay/connector/app/RestClientFactory.java:14]: Looks like this is candidate to move to pay-java-commons

File: src/main/java/uk/gov/pay/connector/paritycheck/LedgerTransaction.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class LedgerTransaction {
```
Comment (kbottla) [inline src/main/java/uk/gov/pay/connector/paritycheck/LedgerTransaction.java:11]: We could possibly match this to classes we have in publicapi. It makes it easy to extract to pay-java-commons later

**Review by kbottla — APPROVED** [review]
Few minor comments towards moving common code to pay-java-comments

File: src/main/java/uk/gov/pay/connector/app/RestClientFactory.java (hunk 1 above)
Comment (jankowiakmaria) [reply src/main/java/uk/gov/pay/connector/app/RestClientFactory.java:14]: Agree

File: src/main/java/uk/gov/pay/connector/paritycheck/LedgerTransaction.java (hunk 2 above)
Comment (jankowiakmaria) [reply src/main/java/uk/gov/pay/connector/paritycheck/LedgerTransaction.java:11]: There is already `TransactionResponse` in connector - which has different meaning.
Worth thinking about it for sure.

**Pull request merged**